docs/content/Deployment/Platform-Docker.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Deploy with Docker
+permalink: /deployment/platforms/docker
+category: Deployment
+subCategory: Platforms
+menuOrder: 1
+---
+
+This guide walks you through deploying Cube.js with Docker.
+
+## Prerequisites
+
+- Docker CLI
+
+## Configuration
+
+Create a Docker Compose stack by creating a `docker-compose.yml`. An example
+stack using BigQuery as a data source is provided below:
+
+<!-- prettier-ignore-start -->
+[[info |]]
+| Using macOS or Windows? Use `CUBEJS_DB_HOST=host.docker.internal` instead of
+| `localhost` if your database is on the same machine.
+<!-- prettier-ignore-end -->
+
+```yaml
+version: '2.2'
+
+services:
+  cube_api:
+    image: cubejs/cube:v%CURRENT_VERSION
```

Comment:
Is there any reason to not recommend running the latest version of Cube.js via the `latest` tag? (Putting `latest` here will also reduce the complexity of this config.)

Reply:
In production, it's not recommended to use the `latest` tag but instead use a specific version number. Perhaps @ovr can confirm?

Reply:
Okay, makes sense!

Reply:
@hassankhan I think we need to use a plugin for %VERSION, something like https://github.com/getsentry/sentry-docs/blob/master/src/gatsby/plugins/gatsby-remark-variables/index.js